src/features/pages/acuity/acuity-feature-guide/data.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import { StaticImage } from 'gatsby-plugin-image'
+import { AcuityFeaturesType } from '../../../components/molecules/image-text-boxes/types'
+
+const deriv_acuity_row_download = 'https://deriv.link/3Vk37Ds'
+const deriv_acuity_eu_download = 'https://deriv.link/3GXrhz6'
```

Comment:
can these url be imported from contants?

Reply:
same

Reply:
ok